content/applications/sales/crm/acquire_leads/convert.rst

```diff
@@ -2,66 +2,120 @@
 Convert leads into opportunities
 ================================
 
-The system can generate leads instead of opportunities, in order to add
-a qualification step before converting a *Lead* into an
-*Opportunity* and assigning to the right sales people.
-
-You can activate this mode from the CRM Settings. It applies to all your sales
-channels by default. But you can make it specific for specific channels
-from their configuration form.
+*Leads* act as qualifying steps before an opportunity is created. This provides additional time
+before a potential opportunity is assigned to a sales person.
 
 Configuration
 =============
 
-For this feature to work, go to :menuselection:`CRM --> Configuration --> Settings`
-and activate the *Leads* feature.
+To activate the *Leads* setting, navigate to :menuselection:`CRM app --> Configuration --> Settings`
+and check the box labeled, :guilabel:`Leads`. Then, click :guilabel:`Save`.
 
-.. image:: convert/convert01.png
+.. image:: convert/convert-leads-leads-setting.png
     :align: center
+    :alt: Leads setting on CRM configuration page.
```

Comment:
Image alignment - the :align: and :alt: attributes need to be inline with the "i" in image 🙂
```suggestion
.. image:: convert/convert-leads-leads-setting.png
   :align: center
   :alt: Leads setting on CRM configuration page.
```